mleap-tensorflow/src/main/scala/ml/combust/mleap/tensorflow/TensorflowTransformerOp.scala

```diff
@@ -71,15 +71,18 @@ class TensorflowTransformerOp extends MleapOp[TensorflowTransformer, TensorflowM
       val outputs = outputNames.zip(outputTypes.zip(outputShapes).map {
         case (b, s) => core.types.DataType(b, s).asInstanceOf[TensorType]
       })
-
-      val graph = new org.tensorflow.Graph()
-      val graphDef = GraphDef.parseFrom(graphBytes)
-      graph.importGraphDef(graphDef)
-      TensorflowModel(graph = Some(graph),
+      val modelBytes = format match {
+        case Some("graph") | None => Files.readAllBytes(context.file("graph.pb"))
+        case Some("saved_model") => Files.readAllBytes(context.file("saved_model.zip"))
+        case _ => throw new UnsupportedOperationException("Only support `saved_model` and `graph` format")
```

Comment:
I don't think this is backward compatible, as bundles serialized before won't have the format set, so in that case, we should just assume `graph`.

Reply:
I think this should be covered by the ` | None` part of the first case

Reply:
ah yes, sorry i didn't see the or part of that case.